trusted/service/sys/types/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! FFI types for the SGX SDK trusted service library (tservice).
+
+#![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
+
+// Nesting to work around clippy warnings, see
+// https://github.com/rust-lang/rust-bindgen/issues/1470
+#[allow(clippy::missing_safety_doc)]
+mod bindings {
+    // include!(..);
+    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+}
+pub use bindings::*;
```

Comment:
IIUC we can fold the `allow(clippy...)` into the top level

```suggestion
#![allow(clippy::missing_safety_doc, non_camel_case_types, non_snake_case, non_upper_case_globals)]

include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
```

Reply:
Sorry, I grabbed the wrong base and was missing my signature so force pushed.
Anyway consolidated and also removed that stray `//include!(..);`